### PR TITLE
Single community

### PR DESCRIPTION
**Pull request by GoodPassiveMan**

### CODE REVIEW

File: app/containers/AccountProvider/saga.js [hunk 1]
```diff
@@ -207,7 +212,14 @@ function* updateRefer(user, eosService) {
     const reward = +convertPeerValueToNumberValue(info.common_reward);
     if (reward) {
       const locale = yield select(makeSelectLocale());
-      setCookie({ name: receivedCookieName, value: true });
+      setCookie({
+        name: receivedCookieName,
+        value: true,
+        options: {
+          domain: '.peeranha.io',
+          expires: 'Tue, 19 Jan 2038 01:14:07 GMT',
```
Comment (sergeyi23) [inline app/containers/AccountProvider/saga.js]: Never expires date can be moved to the constants. It is used multiple times.

File: app/containers/AccountProvider/saga.js [hunk 2]
```diff
@@ -207,7 +212,14 @@ function* updateRefer(user, eosService) {
     const reward = +convertPeerValueToNumberValue(info.common_reward);
     if (reward) {
       const locale = yield select(makeSelectLocale());
-      setCookie({ name: receivedCookieName, value: true });
+      setCookie({
+        name: receivedCookieName,
+        value: true,
+        options: {
+          domain: '.peeranha.io',
```
Comment (sergeyi23) [inline app/containers/AccountProvider/saga.js]: '.peeranha.io` used in multiples places. Needs to be in constants.

File: app/containers/AccountProvider/saga.js [hunk 3]
```diff
@@ -219,6 +231,10 @@ function* updateRefer(user, eosService) {
     setCookie({
       name: noInviterCookieName,
       value: true,
+      options: {
```
Comment (sergeyi23) [inline app/containers/AccountProvider/saga.js]: The same options are passed to `setCookie `. Might be better to pass flag arguments to that function and then let function to set options. E.g. `neverExpires`, `allowSubdomains` boolean arguments.

File: .env.prod [hunk 4]
```diff
@@ -21,3 +21,4 @@ EOS_PORT_DEFAULT='443'
 EOS_CHAINID_DEFAULT='4667b205c6838ef70ff7988f6e8257e8be0e1284a2f59699054a018f743b1d11'
 NEW_ENDPOINT_PERIOD=600000
 APP_LOCATION='https://peeranha.io'
+SINGLE_COMMUNITIES="2,https://ontology.peeranha.io"
```
Comment (sergeyi23) [inline .env.prod]: Single line configuration is hard to manage as we add more communities and add more values to the config. Can we configure name of configuration file here instead?

For example:

`COMMUNITIES_CONFIG_FILE_NAME='communities-config-prod'`

and the in the import:

`import communitiesConfig from process.env.COMMUNITIES_CONFIG_FILE_NAME;`

Comment (GoodPassiveMan) [reply .env.prod]: @sergeyi23 I refacroted cookie utils.
About communities config. I fully agreed with you about one line config. Also it's impossible to use variable as path. So created mapped object at community config file which contains env values as fields and communities as values

**Pull request merged**